### PR TITLE
🌐 [i18n-KO] Translated `model_doc/bart.md` to Korean

### PR DESCRIPTION
**Pull request by fabxoe**
# What does this PR do?

Translated the `model_doc/bart.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [ ] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

@chhaewxn, @ahnjj, @jun048098, @fabxoe, @nuatmochoi, @heuristicwave

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@stevhliu May you please review this PR?

### CODE REVIEW

File: docs/source/ko/model_doc/bart.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+- [`TFBartForConditionalGeneration`]는 이 [예제 스크립트](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/translation)와 [노트북](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/translation-tf.ipynb)을 참고 하세요.
+- [번역 작업 가이드](../tasks/translation)
+
+추가적으로 볼 것들:
+- [텍스트 분류 작업 가이드](../tasks/sequence_classification)
+- [질문 답변 작업 가이드](../tasks/question_answering)
+- [인과적 언어 모델링 작업 가이드](../tasks/language_modeling)
+- 이 [논문](https://arxiv.org/abs/2010.13002)에서는 [증류된 체크포인트](https://huggingface.co/models?search=distilbart)에 대해 설명합니다.
+
+## BartConfig
```
Comment (fabxoe) [inline docs/source/ko/model_doc/bart.md:119]:
```suggestion
## BartConfig[[transformers.BartConfig]]
```

File: docs/source/ko/model_doc/bart.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+
+[[autodoc]] BartConfig
+    - all
+
+## BartTokenizer
```
Comment (fabxoe) [inline docs/source/ko/model_doc/bart.md:124]:
```suggestion
## BartTokenizer[[transformers.BartTokenizer]]
```

File: docs/source/ko/model_doc/bart.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+
+[[autodoc]] BartTokenizer
+    - all
+
+## BartTokenizerFast
```
Comment (fabxoe) [inline docs/source/ko/model_doc/bart.md:129]:
```suggestion
## BartTokenizerFast[[transformers.BartTokenizerFast]]
```

File: docs/source/ko/model_doc/bart.md [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+
+[[autodoc]] BartTokenizerFast
+    - all
+
+
+<frameworkcontent>
+<pt>
+
+## BartModel
```
Comment (fabxoe) [inline docs/source/ko/model_doc/bart.md:138]:
```suggestion
## BartModel[[transformers.BartModel]]
```

File: docs/source/ko/model_doc/bart.md [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+
+[[autodoc]] BartModel
+    - forward
+
+## BartForConditionalGeneration
```
Comment (fabxoe) [inline docs/source/ko/model_doc/bart.md:143]:
```suggestion
## BartForConditionalGeneration[[transformers.BartForConditionalGeneration]]
```

File: docs/source/ko/model_doc/bart.md [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+
+[[autodoc]] BartForConditionalGeneration
+    - forward
+
+## BartForSequenceClassification
```
Comment (fabxoe) [inline docs/source/ko/model_doc/bart.md:148]:
```suggestion
## BartForSequenceClassification[[transformers.BartForSequenceClassification]]
```

File: docs/source/ko/model_doc/bart.md [hunk 7]
```diff
@@ -0,0 +1,220 @@
...
+
+[[autodoc]] BartForSequenceClassification
+    - forward
+
+## BartForQuestionAnswering
```
Comment (fabxoe) [inline docs/source/ko/model_doc/bart.md:153]:
```suggestion
## BartForQuestionAnswering[[transformers.BartForQuestionAnswering]]
```

File: docs/source/ko/model_doc/bart.md [hunk 8]
```diff
@@ -0,0 +1,220 @@
...
+
+[[autodoc]] BartForQuestionAnswering
+    - forward
+
+## BartForCausalLM
```
Comment (fabxoe) [inline docs/source/ko/model_doc/bart.md:158]:
```suggestion
## BartForCausalLM[[transformers.BartForCausalLM]]
```

File: docs/source/ko/model_doc/bart.md [hunk 9]
```diff
@@ -0,0 +1,220 @@
...
+
+[[autodoc]] BartForCausalLM
+    - forward
+
+</pt>
+<tf>
+
+## TFBartModel
```
Comment (fabxoe) [inline docs/source/ko/model_doc/bart.md:166]:
```suggestion
## TFBartModel[[transformers.TFBartModel]]
```

File: docs/source/ko/model_doc/bart.md [hunk 10]
```diff
@@ -0,0 +1,220 @@
...
+
+[[autodoc]] TFBartModel
+    - call
+
+## TFBartForConditionalGeneration
```
Comment (fabxoe) [inline docs/source/ko/model_doc/bart.md:171]:
```suggestion
## TFBartForConditionalGeneration[[transformers.TFBartForConditionalGeneration]]
```

File: docs/source/ko/model_doc/bart.md [hunk 11]
```diff
@@ -0,0 +1,220 @@
...
+
+[[autodoc]] TFBartForConditionalGeneration
+    - call
+
+## TFBartForSequenceClassification
```
Comment (fabxoe) [inline docs/source/ko/model_doc/bart.md:176]:
```suggestion
## TFBartForSequenceClassification[[transformers.TFBartForSequenceClassification]]
```

File: docs/source/ko/model_doc/bart.md [hunk 12]
```diff
@@ -0,0 +1,220 @@
...
+
+[[autodoc]] TFBartForSequenceClassification
+    - call
+
+</tf>
+<jax>
+
+## FlaxBartModel
```
Comment (fabxoe) [inline docs/source/ko/model_doc/bart.md:184]:
```suggestion
## FlaxBartModel[[transformers.FlaxBartModel]]
```

File: docs/source/ko/model_doc/bart.md [hunk 13]
```diff
@@ -0,0 +1,220 @@
...
+
+[[autodoc]] FlaxBartModel
+    - __call__
+    - encode
+    - decode
+
+## FlaxBartForConditionalGeneration
```
Comment (fabxoe) [inline docs/source/ko/model_doc/bart.md:191]:
```suggestion
## FlaxBartForConditionalGeneration[[transformers.FlaxBartForConditionalGeneration]]
```

File: docs/source/ko/model_doc/bart.md [hunk 14]
```diff
@@ -0,0 +1,220 @@
...
+
+[[autodoc]] FlaxBartForConditionalGeneration
+    - __call__
+    - encode
+    - decode
+
+## FlaxBartForSequenceClassification
```
Comment (fabxoe) [inline docs/source/ko/model_doc/bart.md:198]:
```suggestion
## FlaxBartForSequenceClassification[[transformers.FlaxBartForSequenceClassification]]
```

File: docs/source/ko/model_doc/bart.md [hunk 15]
```diff
@@ -0,0 +1,220 @@
...
+
+[[autodoc]] FlaxBartForSequenceClassification
+    - __call__
+    - encode
+    - decode
+
+## FlaxBartForQuestionAnswering
```
Comment (fabxoe) [inline docs/source/ko/model_doc/bart.md:205]:
```suggestion
## FlaxBartForQuestionAnswering[[transformers.FlaxBartForQuestionAnswering]]
```

File: docs/source/ko/model_doc/bart.md [hunk 16]
```diff
@@ -0,0 +1,220 @@
...
+
+[[autodoc]] FlaxBartForQuestionAnswering
+    - __call__
+    - encode
+    - decode
+
+## FlaxBartForCausalLM
```
Comment (fabxoe) [inline docs/source/ko/model_doc/bart.md:212]:
```suggestion
## FlaxBartForCausalLM[[transformers.FlaxBartForCausalLM]]
```

**Review by fabxoe — COMMENTED** [review]
fix: suggestions

File: docs/source/ko/model_doc/bart.md [hunk 17]
```diff
@@ -0,0 +1,220 @@
...
+<div class="flex flex-wrap space-x-1">
+<a href="https://huggingface.co/models?filter=bart">
+<img alt="Models" src="https://img.shields.io/badge/All_model_pages-bart-blueviolet">
+</a>
+<a href="https://huggingface.co/spaces/docs-demos/bart-large-mnli">
+<img alt="Spaces" src="https://img.shields.io/badge/%F0%9F%A4%97%20Hugging%20Face-Spaces-blue">
+</a>
+</div>
+
+## 오버뷰[[overview]]
```
Comment (ahnjj) [inline docs/source/ko/model_doc/bart.md:28]:
```suggestion
## 개요 [[overview]]
```

File: docs/source/ko/model_doc/bart.md [hunk 18]
```diff
@@ -0,0 +1,220 @@
...
+- [마스크 언어 모델링 작업 가이드](../tasks/masked_language_modeling)
+
+<PipelineTag pipeline="translation"/>
+
+- [Seq2SeqTrainer를 이용하여 인도어를 영어로 번역하는 mBART를 미세 조정](https://colab.research.google.com/github/vasudevgupta7/huggingface-tutorials/blob/main/translation_training.ipynb)하는 방법에 대한 가이드. 🌎
+- [`BartForConditionalGeneration`]는 이 [예제 스크립트](https://github.com/huggingface/transformers/tree/main/examples/pytorch/translation)와 [노트북](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/translation.ipynb)을 참고하세요.
+- [`TFBartForConditionalGeneration`]는 이 [예제 스크립트](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/translation)와 [노트북](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/translation-tf.ipynb)을 참고 하세요.
+- [번역 작업 가이드](../tasks/translation)
+
+추가적으로 볼 것들:
```
Comment (ahnjj) [inline docs/source/ko/model_doc/bart.md:113]:
```suggestion
추가 자료:
```

File: docs/source/ko/model_doc/bart.md [hunk 19]
```diff
@@ -0,0 +1,220 @@
...
+
+## 리소스[[resources]]
+
+BART를 시작하는 데 도움이 되는 Hugging Face와 community 자료 목록(🌎로 표시됨) 입니다. 여기에 포함될 자료를 제출하고 싶으시다면 PR(Pull Request)를 열어주세요. 리뷰 해드리겠습니다! 자료는 기존 자료를 복제하는 대신 새로운 내용을 담고 있어야 합니다.
+
+<PipelineTag pipeline="summarization"/>
+
+- [분산형 학습: 🤗 Transformers와 Amazon SageMaker를 이용하여 요약하기 위한 BART/T5 학습](https://huggingface.co/blog/sagemaker-distributed-training-seq2seq)에 대한 블로그 포스트.
+- [blurr를 이용하여 fastai로 요약하기 위한 BART를 미세 조정](https://colab.research.google.com/github/ohmeow/ohmeow_website/blob/master/posts/2021-05-25-mbart-sequence-classification-with-blurr.ipynb)하는 방법에 대한 노트북. 🌎
+- [Trainer 클래스를 사용하여 두 언어로 요약하기 위한 BART를 미세 조정](https://colab.research.google.com/github/elsanns/xai-nlp-notebooks/blob/master/fine_tune_bart_summarization_two_langs.ipynb)하는 방법에 대한 노트북. 🌎
```
Comment (ahnjj) [inline docs/source/ko/model_doc/bart.md:90]:
```suggestion
- [Trainer 클래스를 사용하여 두 가지 언어로 요약하기 위한 BART 미세 조정](https://colab.research.google.com/github/elsanns/xai-nlp-notebooks/blob/master/fine_tune_bart_summarization_two_langs.ipynb)하는 방법에 대한 노트북. 🌎
```
더 자연스러운 표현으로 바꾸었습니다 :)

File: docs/source/ko/model_doc/bart.md [hunk 20]
```diff
@@ -0,0 +1,220 @@
...
+  * 무작위 토큰 마스킹 (BERT 처럼)
+  * 무작위 토큰 삭제
+  * k개 토큰의 범위를 단일 마스크 토큰으로 마스킹 (0개 토큰의 범위는 마스크 토큰의 삽입을 의미)
+  * 문장 순서 뒤섞기
+  * 특정 토큰에서 시작하도록 문서 회전
+
+## 구현 노트[[implementation-notes]]
+
+- Bart는 시퀀스 분류에 `token_type_ids`를 사용하지 않는다. 적절하게 나누기 위해서 [`BartTokenizer`]나
+  [`~BartTokenizer.encode`]를 사용한다.
```
Comment (ahnjj) [inline docs/source/ko/model_doc/bart.md:56]:
```suggestion
  [`~BartTokenizer.encode`]를 사용합니다.
```
다른 부분들은 높임체(?)로 쓰셔서 통일했습니다!

File: docs/source/ko/model_doc/bart.md [hunk 21]
```diff
@@ -0,0 +1,220 @@
...
+- [`BartModel`]의 정방향 전달은 `decoder_input_ids`가 전달되지 않으면 `decoder_input_ids`를 자동으로 생성할 것입니다. 이는 다른 일부 모델링 API와 다른 점이라 할 수 있습니다. 이 기능의 일반적인 사용 사례는 마스크 채우기(mask filling)입니다.
+- Model predictions are intended to be identical to the original implementation when
```
Comment (ahnjj) [inline docs/source/ko/model_doc/bart.md:58]:
```suggestion

```
아래줄에 번역 해두신 것 같아 삭제되어야할 것 같습니다.

File: docs/source/ko/model_doc/bart.md [hunk 22]
```diff
@@ -0,0 +1,220 @@
...
+  `forced_bos_token_id=0`. This only works, however, if the string you pass to
```
Comment (ahnjj) [inline docs/source/ko/model_doc/bart.md:59]:
```suggestion
```
아래줄에 번역 해두신 것 같아 삭제되어야할 것 같습니다.

File: docs/source/ko/model_doc/bart.md [hunk 23]
```diff
@@ -0,0 +1,220 @@
...
+  [`fairseq.encode`] starts with a space.
```
Comment (ahnjj) [inline docs/source/ko/model_doc/bart.md:60]:
```suggestion
```
아래줄에 번역 해두신 것 같아 삭제되어야할 것 같습니다.

**Review by ahnjj — CHANGES_REQUESTED** [review]

File: docs/source/ko/model_doc/bart.md [hunk 24]
```diff
@@ -0,0 +1,220 @@
...
+- Bart는 양방향 인코더(BERT와 유사)와 왼쪽에서 오른쪽으로 디코딩하는 디코더(GPT와 유사)를 사용하는 표준 seq2seq/기계 번역 아키텍처를 사용합니다.
+- 사전 훈련 작업은 원래 문장의 순서를 무작위로 섞고, 텍스트의 일부 구간을 단일 마스크 토큰으로 대체하는 새로운 인필링(in-filling) 방식을 포함합니다.
+- BART는 특히 텍스트 생성을 위한 미세 조정에 효과적이지만 이해 작업에도 잘 작동합니다. GLUE와 SQuAD에서 비슷한 훈련 리소스로 RoBERTa의 성능과 일치하며, 추상적 대화, 질의응답, 요약 작업 등에서 최대 6 ROUGE 점수의 향상을 보이며 새로운 최고 성능을 달성했습니다.
+
+이 모델은 [sshleifer](https://huggingface.co/sshleifer)에 의해 기여 되었습니다. 저자의 코드는 [이곳](https://github.com/pytorch/fairseq/tree/master/examples/bart)에서 확인할 수 있습니다.
+
+## 사용 팁:[[usage-tips]]
+
+- BART는 절대 위치 임베딩을 사용하는 모델이므로 일반적으로 입력을 왼쪽보다는 오른쪽에 패딩하는 것이 좋습니다.
+- 인코더와 디코더가 있는 seq2seq 모델입니다. 인코더에는 손상된 버전의 토큰이 입력되고, 디코더에는 원래 토큰이 입력됩니다(단, 일반적인 트랜스포머 디코더처럼 미래 단어를 숨기는 마스크가 있습니다). 사전 훈련 작업에서 인코더에 적용되는 변환들의 구성은 다음과 같습니다:
```
Comment (jun048098) [inline docs/source/ko/model_doc/bart.md:43]:
```suggestion
- 인코더와 디코더가 있는 seq2seq 모델입니다. 인코더에는 손상된 토큰이(corrupted tokens) 입력되고, 디코더에는 원래 토큰이 입력됩니다(단, 일반적인 트랜스포머 디코더처럼 미래 단어를 숨기는 마스크가 있습니다). 사전 훈련 작업에서 인코더에 적용되는 변환들의 구성은 다음과 같습니다:
```

File: docs/source/ko/model_doc/bart.md [hunk 25]
```diff
@@ -0,0 +1,220 @@
...
+
+  * 무작위 토큰 마스킹 (BERT 처럼)
+  * 무작위 토큰 삭제
+  * k개 토큰의 범위를 단일 마스크 토큰으로 마스킹 (0개 토큰의 범위는 마스크 토큰의 삽입을 의미)
+  * 문장 순서 뒤섞기
+  * 특정 토큰에서 시작하도록 문서 회전
+
+## 구현 노트[[implementation-notes]]
+
+- Bart는 시퀀스 분류에 `token_type_ids`를 사용하지 않는다. 적절하게 나누기 위해서 [`BartTokenizer`]나
```
Comment (jun048098) [inline docs/source/ko/model_doc/bart.md:55]:
```suggestion
- Bart는 시퀀스 분류에 `token_type_ids`를 사용하지 않습니다. 적절하게 나누기 위해서 [`BartTokenizer`]나
```

File: docs/source/ko/model_doc/bart.md [hunk 26]
```diff
@@ -0,0 +1,220 @@
...
+  [`~BartTokenizer.encode`]를 사용한다.
+- [`BartModel`]의 정방향 전달은 `decoder_input_ids`가 전달되지 않으면 `decoder_input_ids`를 자동으로 생성할 것입니다. 이는 다른 일부 모델링 API와 다른 점이라 할 수 있습니다. 이 기능의 일반적인 사용 사례는 마스크 채우기(mask filling)입니다.
```
Comment (jun048098) [inline docs/source/ko/model_doc/bart.md:57]:
```suggestion
- [`BartModel`]의 정방향 전달은 `decoder_input_ids`가 전달되지 않으면 `decoder_input_ids`를 자동으로 생성할 것입니다. 이는 다른 일부 모델링 API와 다른 점입니다. 이 기능의 일반적인 사용 사례는 마스크 채우기(mask filling)입니다.
```

File: docs/source/ko/model_doc/bart.md [hunk 27]
```diff
@@ -0,0 +1,220 @@
...
+- Model predictions are intended to be identical to the original implementation when
+  `forced_bos_token_id=0`. This only works, however, if the string you pass to
+  [`fairseq.encode`] starts with a space.
+  모델 예측은 `forced_bos_token_id=0`일 때 기존 구현과 동일하게 작동하도록 의도되었습니다. 하지만, [fairseq.encode]에 전달하는 문자열이 공백으로 시작할 때만 이 기능이 작동합니다.
```
Comment (jun048098) [inline docs/source/ko/model_doc/bart.md:61]:
```suggestion
  모델 예측은 `forced_bos_token_id=0`일 때 기존 구현과 동일하게 작동하도록 의도되었습니다. 하지만, [`fairseq.encode`]에 전달하는 문자열이 공백으로 시작할 때만 이 기능이 작동합니다.
```

File: docs/source/ko/model_doc/bart.md [hunk 28]
```diff
@@ -0,0 +1,220 @@
...
+- [Seq2SeqTrainer를 이용하여 인도어를 영어로 번역하는 mBART를 미세 조정](https://colab.research.google.com/github/vasudevgupta7/huggingface-tutorials/blob/main/translation_training.ipynb)하는 방법에 대한 가이드. 🌎
+- [`BartForConditionalGeneration`]는 이 [예제 스크립트](https://github.com/huggingface/transformers/tree/main/examples/pytorch/translation)와 [노트북](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/translation.ipynb)을 참고하세요.
+- [`TFBartForConditionalGeneration`]는 이 [예제 스크립트](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/translation)와 [노트북](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/translation-tf.ipynb)을 참고 하세요.
+- [번역 작업 가이드](../tasks/translation)
+
+추가적으로 볼 것들:
+- [텍스트 분류 작업 가이드](../tasks/sequence_classification)
+- [질문 답변 작업 가이드](../tasks/question_answering)
+- [인과적 언어 모델링 작업 가이드](../tasks/language_modeling)
+- 이 [논문](https://arxiv.org/abs/2010.13002)에서는 [증류된 체크포인트](https://huggingface.co/models?search=distilbart)에 대해 설명합니다.
```
Comment (jun048098) [inline docs/source/ko/model_doc/bart.md:117]:
```suggestion
- 이 [논문](https://arxiv.org/abs/2010.13002)은 [증류된 체크포인트](https://huggingface.co/models?search=distilbart)에 대해 설명합니다.
```

**Review by fabxoe — COMMENTED** [review]
fix: resolve suggestions

**Review by fabxoe — COMMENTED** [review]
fix: resolve suggestions

File: docs/source/ko/model_doc/bart.md [hunk 29]
```diff
@@ -0,0 +1,217 @@
...
+  * k개 토큰의 범위를 단일 마스크 토큰으로 마스킹 (0개 토큰의 범위는 마스크 토큰의 삽입을 의미)
+  * 문장 순서 뒤섞기
+  * 특정 토큰에서 시작하도록 문서 회전
+
+## 구현 노트[[implementation-notes]]
+
+- Bart는 시퀀스 분류에 `token_type_ids`를 사용하지 않습니다. 적절하게 나누기 위해서 [`BartTokenizer`]나
+  [`~BartTokenizer.encode`]를 사용합니다.
+- [`BartModel`]의 정방향 전달은 `decoder_input_ids`가 전달되지 않으면 `decoder_input_ids`를 자동으로 생성할 것입니다. 이는 다른 일부 모델링 API와 다른 점입니다. 이 기능의 일반적인 사용 사례는 마스크 채우기(mask filling)입니다.
+  모델 예측은 `forced_bos_token_id=0`일 때 기존 구현과 동일하게 작동하도록 의도되었습니다. 하지만, [`fairseq.encode`]에 전달하는 문자열이 공백으로 시작할 때만 이 기능이 작동합니다.
```
Comment (stevhliu) [inline docs/source/ko/model_doc/bart.md:58]: I think this should be two separate items here.
```suggestion
- [`BartModel`]의 정방향 전달은 `decoder_input_ids`가 전달되지 않으면 `decoder_input_ids`를 자동으로 생성할 것입니다. 이는 다른 일부 모델링 API와 다른 점입니다. 이 기능의 일반적인 사용 사례는 마스크 채우기(mask filling)입니다.
-  모델 예측은 `forced_bos_token_id=0`일 때 기존 구현과 동일하게 작동하도록 의도되었습니다. 하지만, [`fairseq.encode`]에 전달하는 문자열이 공백으로 시작할 때만 이 기능이 작동합니다.
```

**Review by stevhliu — COMMENTED** [review]
Thanks! Just one suggestion, otherwise LGTM 👍

**Review by stevhliu — APPROVED** [review]
👍

**Pull request merged**